resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -41,22 +50,30 @@ export default () => {
         clearFlashes('files');
         setSelectedFiles([]);
         setDirectory(hashToPath(hash));
-    }, [ hash ]);
+    }, [hash]);
```

Comment:
Add back the spaces

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3926/commits/d09749a8dc1812985035c2e249de4a6ff6b1effd